peter-piper.js

```diff
@@ -19,7 +19,10 @@ peterpiper += "Where's the peck of pickled peppers Peter Piper picked?";
 
 var counter = 0;
 
-// TODO: Count the number of times an uppercase P or lowercase p character
-//       occurs in the `peterpiper` string.
+for (var i = 0; i < peterpiper.length; i++) {
+  if(peterpiper.charAt(i) == 'p') {
```

Comment:
It looks like you are only checking for lowercase 'p's. You should either convert peterpiper to lowercase before looping through the characters, OR add an "logical OR" check, and check for the uppercase 'P' character also.